sources/web/datalab/polymer/components/resizable-divider/resizable-divider.html

```diff
@@ -0,0 +1,80 @@
+<!--
+Copyright 2017 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+or implied. See the License for the specific language governing permissions and limitations under
+the License.
+-->
+
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<dom-module id="resizable-divider">
+  <template>
+
+    <style include="datalab-shared-styles">
+      :host {
+        --divider-width: 5px;
+      }
+      #container {
+        height: 100%;
+        width: 100%;
+        display: flex;
+        position: relative;
+      }
+      .pane {
+        overflow: auto;
+        width: 50%;
+        user-select: none;
+      }
+      /**
+       * Slotted elements should fill the content slots, and be border-boxed so
+       * they can add padding.
+       */
+      ::slotted(*) {
+        width: 100%;
+        height: 100%;
+        box-sizing: border-box;
+      }
+      /**
+      * Consider applying a style rule that can be passed to this element.
+      */
+      #divider {
+        cursor: col-resize;
+        user-select: none;
+        position: absolute;
+        top: 0px;
+        bottom: 0px;
+        left: calc(50% - var(--divider-width));
+        right: calc(50% + var(--divider-width));
+        width: calc(2 * var(--divider-width));
+      }
+      #divider.active {
+        user-select: none;
+        background-color: var(--selection-bg-color);
+      }
+    </style>
+
+    <div id="container">
+
+      <div id="p1" class="pane">
```

Comment:
Sure, I can make the names clearer just because it's helpful to do that.
If the id were "left-pane" and I inserted a div with the same id as one of the panes, it appears in a different DOM altogether, so you'd reference them differently in the code. The one inside the divider element is in its shadow DOM, and you get a reference to it by calling `this.$.id`, and the one that gets inserted into the slot is in the light DOM, you can get it by calling `this.querySelector('#id')`.
In some sense, you can say these two elements are "different as night and day" :)